src/ValueFormatters/QuantityFormatter.php

```diff
@@ -63,14 +63,25 @@ class QuantityFormatter extends ValueFormatterBase {
 	private $vocabularyUriFormatter;
 
 	/**
+	 * @var string
+	 */
+	private $quantityWithUnitFormat;
+
+	/**
+	 * @since 0.6
+	 *
+	 * @param FormatterOptions|null $options
 	 * @param DecimalFormatter|null $decimalFormatter
 	 * @param ValueFormatter|null $vocabularyUriFormatter
-	 * @param FormatterOptions|null $options
+	 * @param string|null $quantityWithUnitFormat Warning, this must be under the control of the
+	 * application, not under the control of the user, because it allows HTML injections in
+	 * subclasses that return HTML.
 	 */
 	public function __construct(
+		FormatterOptions $options = null,
```

Comment:
You merged it: https://github.com/DataValues/Number/pull/41/files#diff-8c77e0fb91ce0037fa8e2fd7b88a35ab